packages/itwinui-css/backstop/tests/menu.html

```diff
@@ -35,10 +35,6 @@
         padding: 12px;
       }
 
-      #demo-scrollable {
-        --iui-menu-max-height: 96px;
-      }
```

Comment:
Removing these lines creates small changes in 10 tests. Four are expected because they involve the new menu items on [L148-224](https://github.com/iTwin/iTwinUI/pull/777/files#diff-4d20b96669e20969ecd169274440b0f0a55f12d3fec73826fccc3285324b8261R148-R224):
* iTwinUI_menu_Type_Default_0_demo-default_0_.png
  * ![image](https://user-images.githubusercontent.com/45748283/194637939-d832c74c-a064-48fe-b2e2-8028abc5f7f6.png)
* iTwinUI_menu_State_hover__focus_0_demo-default_0_.png
  * ![image](https://user-images.githubusercontent.com/45748283/194638040-f3ab5429-b531-41a1-a838-25fcdeb7eade.png)
* iTwinUI_dark_menu_Type_Default_0_demo-default_0_.png
  * ![image](https://user-images.githubusercontent.com/45748283/194638147-6fc02975-4c69-4e96-b53a-6d966df995ae.png)
* iTwinUI_dark_menu_State_hover__focus_0_demo-default_0_.png
  * ![image](https://user-images.githubusercontent.com/45748283/194638794-c016b9bc-7191-4cd5-a070-27a4ad922aa8.png)

All the other 6 image changes are unintended. Why could this be happening? Can we approve the other 6 images too?

Reply:
**[RESOLVED]** As discussed with the team, these small pixel changes happen sometimes. Thus, I think we can ignore the 6 unintended changes and approve them.